cmd/tsconnect/common.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strconv"
+
+	esbuild "github.com/evanw/esbuild/pkg/api"
+)
+
+type EntryPointMap = map[string]string
```

Comment:
With a `=` this is a type alias. Type aliases are almost never what you want.

I'd just drop it. And then document this a bit. What are the keys/values?

Reply:
Ended up dropping this preprocessing altogether, and thus the need for this type. I now persist the entire esbuild metadata JSON, and build this map at serving time (since we only need to do it once, during server startup).